lib/less-parser.js

```diff
@@ -164,7 +164,12 @@ export default class LessParser extends Parser {
         break;
       }
       else if (tokn[0] === 'brackets') {
-        directives.push(tokn);
+        if (node.urlFunc) {
+          node.importPath = tokn[1].replace(/[()]/g, '');
```

Comment:
Not sure if we wanna remove the `()` here or in the stringifier? I noticed that they're inserted there anyway but removing them here makes it a bit easier to work with the AST from other tools. And technically, `()` aren't part of the URL even though we include the quotes in `importPath` when they exist.

Anyway, I don't have a strong opinion on either, just wanted to point it out.